packages/wrangler/src/api/integrations/platform/index.ts

```diff
@@ -103,27 +111,37 @@ export async function getPlatformProxy<
 >(
 	options: GetPlatformProxyOptions = {}
 ): Promise<PlatformProxy<Env, CfProperties>> {
+	const experimentalRemoteBindings = !!options.experimental?.remoteBindings;
+
 	const env = options.environment;
 
 	const rawConfig = readConfig({
 		config: options.configPath,
 		env,
 	});
 
-	const miniflareOptions = await run(
-		{
-			MULTIWORKER: false,
-			RESOURCES_PROVISION: false,
-			// TODO: when possible remote bindings should be made available for getPlatformProxy
-			REMOTE_BINDINGS: false,
-		},
-		() => getMiniflareOptionsFromConfig(rawConfig, env, options)
+	let remoteProxySession: RemoteProxySession | undefined = undefined;
+	if (experimentalRemoteBindings && rawConfig.configPath) {
+		const a = await maybeStartOrUpdateRemoteProxySession(
+			rawConfig.configPath,
+			null
+		);
+		remoteProxySession = a?.session;
+	}
+
+	const miniflareOptions = await getMiniflareOptionsFromConfig(
+		rawConfig,
+		env,
+		options,
+		remoteProxySession?.remoteProxyConnectionString
 	);
 
 	const mf = new Miniflare({
 		script: "",
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore
```

Comment:
Can you make this a `@ts-expect-error` with an explanatory comment?

Reply:
good call 🙂👍

Reply:
done 🙂